src/components/Header/index.tsx

```diff
@@ -35,13 +36,23 @@ import { useWeb3React } from '@web3-react/core';
 import { getErrorMessage } from '../../lib/web3-react';
 import { Button, Avatar, Image } from '@chakra-ui/react';
 import { useStore } from '../../store/index';
+import { NoEthereumProviderError } from '@web3-react/injected-connector';
 
 export const Header = observer(() => {
   const { isOpen: isMobileNavOpen, onToggle } = useDisclosure();
   const { colorMode, toggleColorMode } = useColorMode();
   const { error } = useWeb3React();
   const { lang } = useStore();
-
+  function IsPC(){
```

Comment:
please wrap this code to the `lib/helper.ts`

Reply:
OK